testpar/t_bigio.c

```diff
@@ -1151,14 +1151,14 @@ single_rank_independent_io(void)
         VRFY_G((mspace_id >= 0), "H5Screate_simple mspace_id succeeded");
 
         /* Write data */
-        H5Dwrite(dset_id, H5T_NATIVE_INT, mspace_id, fspace_id, H5P_DEFAULT, data);
+        ret = H5Dwrite(dset_id, H5T_NATIVE_INT, mspace_id, fspace_id, H5P_DEFAULT, data);
         VRFY_G((ret >= 0), "H5Dwrite succeeded");
```

Comment:
ret had never been initialized or set in this test file, so it's very possible t_bigio might've had sporadic failures from checking an uninitialized ret.